.github/workflows/ci.yml

```diff
@@ -355,6 +395,12 @@ jobs:
       with:
         name: ${{ steps.debian-package.outputs.DPKG_NAME }}
         path: ${{ steps.debian-package.outputs.DPKG_PATH }}
+    
+    - name: "Artifact upload: Debian package"
```

Comment:
```suggestion
    - name: "Artifact upload: VSCode extension"
```

Reply:
I think you need to move this task to the vscode_extension_package job. It currently runs for the matrix build, which then fails to upload anything:

https://github.com/sharkdp/numbat/actions/runs/12576115802/job/35051885317?pr=659#step:16:17

Reply:
I think you are right. I was playing around with it a bit to understand the steps and what what should go. I seem to have missed moving this with the rest.